p2p/protocol/circuitv1/relay/relay.go

```diff
@@ -262,6 +265,29 @@ func (r *Relay) handleHopStream(s network.Stream, msg *pb.CircuitRelay) {
 	go r.relayConn(bs, s, dest.ID, src.ID, done)
 }
 
+func (r *Relay) addConn(p peer.ID) {
+	conns := r.conns[p]
+	conns++
+	r.conns[p] = conns
```

Comment:
Is this to set a limit on the tag value?

Reply:
yeah.

Reply:
we can also do a (very small) fixed tag value if you prefer.

Reply:
Upon further thought, I think I'll make it a fixed tag with a value 2; base utility is to distinguish idle connections from connections that have hop circuits on them.

Reply:
done.